### PR TITLE
Mazda: disable hud alerts until we find a way to silence audible alerts 

### PR DESCRIPTION
**Pull request by Jafaral**
Signed-off-by: Jafar Al-Gharaibeh <to.jafar@gmail.com>

### CODE REVIEW

File: selfdrive/car/mazda/mazdacan.py [hunk 1]
```diff
@@ -65,6 +65,8 @@ def create_steering_control(packer, car_fingerprint, frame, apply_steer, lkas):
 
 def create_alert_command(packer, cam_msg: dict, ldw: bool, steer_required: bool):
   values = copy.copy(cam_msg)
+  # TODO: disable alerts for now until we find a way to silence audible warnings
+  steer_required = False
```
Comment (adeebshihadeh) [inline selfdrive/car/mazda/mazdacan.py:69]: this should go in the carcontroller

Comment (Jafaral) [reply selfdrive/car/mazda/mazdacan.py]: sounds good

**Comment by adeebshihadeh** [comment]
I’ll merge this for now since I agree that it’s super annoying, but I expect this will be addressed soon. I don’t care so much about the actual HUD alert, the most important thing is the dash alert.

**Comment by Jafaral** [comment]
> I’ll merge this for now since I agree that it’s super annoying, but I expect this will be addressed soon. I don’t care so much about the actual HUD alert, the most important thing is the dash alert.

I don't disagree @adeebshihadeh. BTW, I just pushed a commit to enable hud alerts for steer lockout

**Comment by adeebshihadeh** [comment]
I think this makes more sense.

**Pull request merged**